### ISSUE TITLE
Suggestion, Add footer partial

### ISSUE THREAD

**Issue by jordibares** [alex-shpak/hugo-book#36]
It would be great to have a footer available as this is handy to me for debugging purposes, as in injecting a debug footer when required.

A structure a bit like this is what I mean... thanks
```
<!DOCTYPE html>
<html>
  {{- partial "head.html" . -}}
  <body>
    {{- partial "header.html" . -}}
    {{- block "main" . }}{{- end }}
    {{- partial "footer.html" . -}}
  </body>
</html>
```

**Comment by alex-shpak** [comment]
Hi!
you can use `docs/inject/body.html` partial for that specific case.
It's injected just before the body closing tag.

**Comment by jordibares** [comment]
Thanks Alex, if you don't mind I will try to add it to my repo and pull a request, I hope I get it right.

**Comment by alex-shpak** [comment]
Resolved via #38
